doc/topics/jinja/index.rst

```diff
@@ -985,7 +985,7 @@ Jinja release older than version 2.9 installed. If Jinja 2.9 or newer is
 installed, then the upstream version of the filter will be used. See the
 `upstream docs`__ for more information.
 
-.. __: http://jinja.pocoo.org/docs/2.10/templates/#tojson
+.. __: https://jinja.palletsprojects.com/en/2.11.x/templates/#tojson
```

Comment:
does jinja have a url that will always point to the latest? is there a reason we are pointing to a specific version?

Reply:
If we pin to a specific jinja version then we should probably link to that version in the docs. Otherwise, latest would be bestest 😂

Reply:
Based on the response by @waynew I'll be leaving it pinned to 2.11.x